### DISCUSSION TITLE
xDrip not connecting to Dexcom Share

### DISCUSSION THREAD

**Discussion by lrussell6868** [Navid200/xDrip#158] [Dexcom]
I was using Nightscout to pull data from my DD to have on my phone and Garmin watch. Something happened and I could not longer get data so I moved to using SugarMate to pull data from Dexcom Share to have data on my watch and a widget on my phone. When that was no longer an option. I started the process of taking advantage of Nightscout (from scratch - i deleted all previous Nightscout work in github and heroku).
I started from scratch following the directions for the classic DIY installation. I have accounts in Github, Heroku and MongoDBatlas. I have Nightscout working and Dexcom share is working. I cannot figure out how to get data to my Garmin Watch and a widget for my phone. I downloaded xDrip.+ I have the most current version and I have followed all of the directions and read suggestions from the facebook pages but I cannot get xDrip+ to pull the data. I an getting the following error: 10:50" Share login erroe" Publisher account password failed code:500.
I have verified my Dexcom username and password are correct and I believe i have all of the settings correct (based on research from previous posts). I seem to be so close but no data.
Thank you!

**Comment by Navid200** [comment]
> I was using Nightscout to pull data from my DD ...

What is DD?

**Reply by lrussell6868** [reply]
Diabetic Daughter

**Reply by Navid200** [reply]
How was she sending data to Nightscout?
